src/pudl_archiver/archivers/epacems.py

```diff
@@ -90,26 +35,28 @@ async def get_resources(self) -> ArchiveAwaitable:
             resjson = file_list.content.decode("utf8").replace("'", '"')
             file_list.close()  # Close connection.
             bulk_files = json.loads(resjson)
-            hourly_state_emissions_files = [
+            quarterly_emissions_files = [
                 file
                 for file in bulk_files
                 if (file["metadata"]["dataType"] == "Emissions")
                 and (file["metadata"]["dataSubType"] == "Hourly")
-                and ("stateCode" in file["metadata"])
+                and ("quarter" in file["metadata"])
                 and (self.valid_year(file["metadata"]["year"]))
             ]
-            logger.info(f"Downloading {len(hourly_state_emissions_files)} total files.")
-            for i, cems_file in enumerate(hourly_state_emissions_files):
-                yield self.get_state_year_resource(file=cems_file, request_count=i)
+            logger.info(f"Downloading {len(quarterly_emissions_files)} total files.")
+            for i, cems_file in enumerate(quarterly_emissions_files):
+                yield self.get_quarter_year_resource(
+                    file=cems_file, request_count=i + 1
+                )  # Count files starting at 1 for human legibility.
         else:
             raise AssertionError(
                 f"EPACEMS API request did not succeed: {file_list.status_code}"
             )
 
-    async def get_state_year_resource(
+    async def get_quarter_year_resource(
```

Comment:
Saying "quarter_year" rather than "year_quarter" (which is the ordering in the filenames and ISO8601) seems a little confusing / unnatural.